core/src/main/scala/chisel3/Printable.scala

```diff
@@ -169,6 +177,8 @@ sealed abstract class FirrtlFormat(private[chisel3] val specifier: Char) extends
   def unpack(ctx: Component): (String, Iterable[String]) = {
     (s"%$specifier", List(bits.ref.fullName(ctx)))
   }
+
+  def unpack_args(): Seq[Bits] = Seq(bits)
```

Comment:
```suggestion
  def unpackArgs(): Seq[Bits] = List(bits)
```
`Seq.apply` does create a `List`, but we should be concrete (note that this codebase very frequently uses `Seq(...)` to construct things but we should stop doing that).